src/nxdoc/nuxeo-server/administration/opensearch-setup.md

```diff
@@ -0,0 +1,963 @@
+
+This page shows you how to prevent the heap to be swapped. Swapping can dramatically decrease performance and stability, so you should ensure it is disabled on production clusters.
+
+It also gives guidance regarding the JVM configuration values like below, and how to avoid duplication and unexpected behaviors:
+```
+OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m
+```
+
+To reduce disk IO you should consider changing the default [translog](https://www.elastic.co/guide/en/elasticsearch/reference/6.3/index-modules-translog.html)
+durability from `request` to `async`.
+Since Nuxeo 10.3 this can be done from `nuxeo.conf`:
+```
+elasticsearch.index.translog.durability=async
+```
+
+If your indices are already created you need some manual operation to change the translog:
+```bash
+curl -H "Content-Type: application/json" -XPUT "https://localhost:9200/nuxeo-uidgen/_settings" -d '{
+  "index.translog.durability" : "async"
+}'
+
+curl -H "Content-Type: application/json" -XPUT "https://localhost:9200/nuxeo-audit/_settings" -d '{
+  "index.translog.durability" : "async"
+}'
+
+curl -H "Content-Type: application/json" -XPUT "https://localhost:9200/nuxeo/_settings" -d '{
+  "index.translog.durability" : "async"
+}'
+
+```
+
+### Setting up the OpenSearch cluster
+
+{{#> callout type='tip' }}
+Use an explicit cluster name by setting the `cluster.name` on each node, in the `opensearch.yml` that can be found:
+- `/usr/share/opensearch/config/opensearch.yml` (Docker)
+- `/etc/opensearch/opensearch.yml` (most Linux distributions)
+{{/callout}}
+
+Extended info about cluster settings can be found in the [OpenSearch documentation](https://opensearch.org/docs/1.3/tuning-your-cluster/index/)
+
+## Configuring Nuxeo to Access the OpenSearch Cluster
+
+Nuxeo supports two protocols to access the OpenSearch cluster: the transport client protocol and the [Rest client](https://opensearch.org/docs/1.3/clients/java-rest-high-level/).
+
+### The REST Client (default)
+
+{{#> callout type='warning' }}
+The OpenSearch Java high-level REST client will be deprecated starting with OpenSearch version 3.0.0 and will be removed in a future release. OpenSearch recommends switching to the Java client instead.
+{{/callout}}
+
+This protocol is supported since Nuxeo 9.3:
+```
+elasticsearch.client=RestClient
+elasticsearch.addressList=https://somenode:9200,https://anothernode:443
+```
+Where:
+- `elasticsearch.client` choose the RestClient protocol
+- `elasticsearch.addressList` is a comma separated list of URL.
+
+### The Transport Client protocol
+
+Here are the `nuxeo.conf` options available for the Transport Client protocol:
+```
+elasticsearch.client=TransportClient
+elasticsearch.addressList=somenode:9300,anothernode:9300
+elasticsearch.clusterName=opensearch
```

Comment:
the clusterName is used for embedded node an afaik don't need to be set to opensearch,
for the rest client it is not used so the doc should be exactly the same as for elastic and should not be duplicated

Reply:
removed the whole page